ui/src/redux/reducers/policies.js

```diff
@@ -156,6 +157,19 @@ export const policies = (state = {}, action) => {
                 draft.expiry = getExpiredTime();
             });
         }
+        case UPDATE_TAGS_TO_STORE: {
+            const { collectionName, collectionWithTags, category } = payload;
```

Comment:
whats the default value of collectionWithTags ?

Reply:
it doesnt have a default value but it expected to be the resource that been updated with the new tags values